internal/kafka/internal/presenters/kafka.go

```diff
@@ -52,9 +53,17 @@ func PresentKafkaRequest(kafkaRequest *dbapi.KafkaRequest) public.KafkaRequest {
 		InstanceType:            kafkaRequest.InstanceType,
 		ReauthenticationEnabled: kafkaRequest.ReauthenticationEnabled,
 		KafkaStorageSize:        kafkaRequest.KafkaStorageSize,
+		BrowserUrl:              fmt.Sprintf("%sapplication-services/streams/kafkas/%s/dashboard", validateBrowserUrl(browserUrl), reference.Id),
```

Comment:
I think this bit will never change: `application-services/streams/kafkas/`. We can move this to the saas template instead. 

```suggestion
		BrowserUrl:              fmt.Sprintf("%s%s/dashboard", validateBrowserUrl(browserUrl), reference.Id),
```